src/sdl2_wrapper.cpp

```diff
@@ -551,6 +567,7 @@ CAMLprim value Val_SDL_Event(SDL_Event *event) {
       v = Val_SDL_WindowEvent(23, event->window.windowID);
       break;
     default:
+      fprintf(stderr, "Unknown event in sdl2_wrapper, event enum code was %d\n", event->type);
```

Comment:
We should remove this as well - might cause issues in the case where we run on Windows w/o a console allocated (we could hook into the `SDL_Log` mechanism though instead)